src/arch/ArchHooks/ArchHooks_Win32Static.cpp

```diff
@@ -22,30 +22,35 @@ static bool g_bTimerInitialized;
  * QueryPerformanceCounter and QueryPerformanceFrequency expect
  * a LARGE_INTEGER, which is a union. These functions store data
  * in the QuadPart of the LARGE_INTEGER, which is a 64-bit integer. */
-LARGE_INTEGER g_liFrequency, g_liStartTime, g_liCurrentTime;
+namespace {
+    LARGE_INTEGER g_liFrequency;
+    LARGE_INTEGER g_liCurrentTime;
+}
```

Comment:
add a `}  // namespace` comment at the end;

Reply:
Sure thing